docs/source/05_solver_config_hooks.rst

```diff
@@ -0,0 +1,60 @@
+.. _solver_config_hooks-section:
+
+Solver Configuration Hooks
+==========================
+
+The solver configuration hooks allow plugins configure internal settings from the ALFAsim.
```

Comment:
+to, -the:
The solver configuration hooks allow plugins *to* configure internal settings from ALFAsim.

+it:
To add an configuration hook *it* is necessary to...